api/rest/client.go

```diff
@@ -0,0 +1,159 @@
+package rest
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/sirupsen/logrus"
+)
+
+// Client is an interface that can talk with dcos-diagnostics REST API and manipulate remote bundles
+type Client interface {
+	// Create ask node url to start bundle creation process with given id
+	Create(ctx context.Context, node string, id string) (*Bundle, error)
+	// Status returns status of bundle with given id on node at the given url
+	Status(ctx context.Context, node string, id string) (*Bundle, error)
+	// GetFile downloads bundle file of bundle with given id from node at the given
+	// url and save it to local filesystem. It returns path where file is stored or
+	// error if there were a problem.
+	GetFile(ctx context.Context, node string, id string) (path string, err error)
```

Comment:
We should allow user to define where bundle should be stored
```suggestion
	GetFile(ctx context.Context, node string, id string, output string) (path string, err error)
```